mod_ci/controllers.py

```diff
@@ -935,14 +935,12 @@ def start_ci():
                     g.db.add(progress)
                     g.db.commit()
                     # If test run status exists, mark them as cancelled
+                    gh_commit = repository.get_commit(test.commit)
                     for status in repository.get_commit(test.commit).get_statuses():
```

Comment:
The new line "gh_commit=..." should come before the comment line "# If test...", right?

Reply:
Oh yes, thanks.
I would change this.